client/my-sites/plans/jetpack-plans/plan-renewal-message/index.tsx

```diff
@@ -1,7 +1,7 @@
 import { useTranslate } from 'i18n-calypso';
 import React, { FunctionComponent } from 'react';
 
-type Props = {};
+type Props = Record< never, never >;
 
 const PlanRenewalMessage: FunctionComponent< Props > = () => {
```

Comment:
We shouldn't need to make empty props:

```suggestion
const PlanRenewalMessage: FunctionComponent = () => {
```

Or even:

```suggestion
const PlanRenewalMessage = (): JSX.Element => {
```